media-video/shotcut/shotcut-19.12.31.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic qmake-utils xdg
+
+DESCRIPTION="A free, open source, cross-platform video editor"
+HOMEPAGE="https://www.shotcut.org/"
+SRC_URI="https://github.com/mltframework/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE=""
+
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+"
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5
+	dev-qt/qtgraphicaleffects:5
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtopengl:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtquickcontrols:5[widgets]
+	dev-qt/qtsql:5
+	dev-qt/qtwebkit:5
+	dev-qt/qtwebsockets:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	media-libs/ladspa-sdk
+	media-libs/libsdl:0
+	media-libs/libvpx
+	>=media-libs/mlt-6.16.0-r1[ffmpeg,frei0r,qt5,sdl,xml]
+	media-libs/webvfx
+	media-libs/x264
+	media-plugins/frei0r-plugins
+	media-sound/lame
+	media-video/ffmpeg
```

Comment:
Many of these deps has a soname, so can you check whether they get linked? Ie do you need a subslot binder `:=` on them?

Reply:
I'll check.

Reply:
Please do, but exclude `dev-qt/*` from that. Subslot operator use on Qt packages is only valid for use of private header (QPA) files subject to API changes.

Reply:
Dropped all libs which are not linked (not found in `ldd /usr/bin/shotcut`):

media-libs/ladspa-sdk (used over media-libs/mlt[jack])
media-libs/libsdl:0 (used over media-libs/mlt[sdl])
media-plugins/frei0r-plugins (used over media-libs/mlt[frei0r])
virtual/jack (used over media-libs/mlt[jack])

media-libs/libvpx (media-video/ffmpeg[vpx] but is not hard requirement)
media-libs/x264 (media-video/ffmpeg[x264] but is not hard requirement)

media-sound/lame